configs/FLYWOOF745AIOV2/config.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F745
+
+#define BOARD_NAME        FLYWOOF745AIOV2
+#define MANUFACTURER_ID   FLWO
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
```

Comment:
There are at least some versions of the "V2" FC with the MPU6000. Suggest retaining the define. 
See: https://discord.com/channels/868013470023548938/1179625370392219769
```suggestion
#define USE_ACC_SPI_MPU6000
#define USE_ACC_SPI_ICM42688P
#define USE_GYRO
#define USE_GYRO_SPI_MPU6000
#define USE_GYRO_SPI_ICM42688P
```

Reply:
FLYWOOF745AIO uses MPU6000 - users on V2 should use this target instead (for correct IMU alignment)
FLYWOOF745AIOV2 uses ICM42688P

Reply:
Ah, great. Looks like the issue I linked above was resolved in https://github.com/betaflight/config/pull/425